src/components/Home.js

```diff
@@ -31,8 +31,8 @@ const Home = () => {
 
   return (
     <div>
-      <BookForm onAdd={handleAddBook} />
       <BookList books={localBooks} onDelete={handleRemoveBook} />
+      <BookForm onAdd={handleAddBook} />
```

Comment:
- [ ] Great job so far but I think here the nav is different than the given design so I highly recommend fixing the logo since the color is blue not black so please fix it to be the same and the size for the logo is bigger than the nav links beside it so please take a look to the given design and try to make it the same 🙏🏻🚀✅

- [ ] KIndly add some margin/padding between the logo and the nav links to make it like the given design ✨🔥

- Your design
![image](https://github.com/Eccb7/book-store/assets/58553711/da67471f-30d6-4d86-9080-0d0915498683)


- given
![image](https://github.com/Eccb7/book-store/assets/58553711/c9f6d28d-3977-4c84-aaf0-d5ed5cfcfff9)